docs/marketplace/dev/front-end/ui-components-library/data-transformers/object-map.md

```diff
@@ -37,22 +42,15 @@ In the example below, the `datasource` will return an array with the transformed
       },
     },
   }"
-></spy-select>
+>
+</spy-select>
 ```
 
-## Interfaces
-
+## Service registration
 
-
-`mapProps`—a Data Transformer that is set up with a configuration object.  
-`propName`—the name of the property from which the value needs to be transformed.
```

Comment:
Should we also keep the `propName` description?

Reply:
Reverted